### DISCUSSION TITLE
USB tethering and transfer files

### DISCUSSION THREAD

**Discussion by owaid** [Mygod/VPNHotspot#503] [Q&A]
Hello

First of thanks to all who contributed to this project.

Is there any solution to make the USB tethering and transfer files between phone and PC at the same time.

Thank you

**Comment by Mygod** [comment]
In USB tethering mode, you can transfer files by running a file server. This could be achieved by apps like [MiXplorer](https://mixplorer.com/).

**Reply by owaid** [reply]
Is there now any free ver
